.github/workflows/flake8.yml

```diff
@@ -0,0 +1,25 @@
+name: Python Flake8 Checks
```

Comment:
why not just use the superlinter as with other repos? it will complain about more things :)

Reply:
Switched to super-linter. It's true it complains a alot. :-) Combinantion of templating with yaml/k8s seems problematic, but many linting issues was resolved.